hack/release-notes.tmpl

```diff
@@ -0,0 +1,18 @@
+# Changelog since {{ .PreviousRevision }}
+
+{{with .NotesWithActionRequired -}}
```

Comment:
How do we use this?

Reply:
These are the steps:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md
3. If no release note is required, just write "NONE".